src/kw_env.sh

```diff
@@ -16,6 +16,7 @@ declare -ga config_file_list=(
 )
 
 declare -gr ENV_CURRENT_FILE='env.current'
+declare -gr ENV_DIR='envs'
```

Comment:
We will need to use this inside build.sh and deploy.sh, let's move it to the `kw` file.

Reply:
But if it's inside the `kw` how could the kw_env_test `include` it?